packages/create-docusaurus/src/index.ts

```diff
@@ -319,4 +317,11 @@ We recommend that you begin by typing:
 
 Happy building awesome websites!
 `);
+  } else {
+    console.error(chalk.red('Dependency installation failed.'));
+    console.log(`The site directory has been created, and you can retry by typing:
+
+  ${chalk.cyan('cd')} ${cdpath}
+  ${chalk.cyan(`${pkgManager} install`)}`);
```

Comment:
Is it a good idea to print just `yarn` if `pkgManager===yarn` because `yarn install` doesn't make much sense.

Reply:
`yarn install` makes sense. In fact I always prefer `yarn install` to `yarn`.

Reply:
Ohh! That's good then. I just read `yarn` instead of `yarn install` elsewhere, that's why I commented to ask. Awesome and sorry for nitpicking; was just looking at the PR code haha :)

Reply:
Yeah, thanks for the review still!

Reply:
Also, Josh; if there are similar beginner friendly issues; always feel free to assign and tag me! I would love to contribute too ;)

Reply:
We are focused on shipping the rc release so the remaining few tasks in our todo list are getting increasingly harder :/ I'll continue to delegate easier issues to the community as soon as I can think of one, so just keep updated